Feature/barebones report

wfulp opened the pull request:
Creating template with no writing and minimal R code. Feel free to cut anything else out you would like.

This resolves #45

Also fixed a bug in the skeleton in generic template.

monicagerber commented:
Hi Jimmy, thanks for making this feature!

I made a couple of changes:
1) Updated vignette so it describes how to use the empty report.
2) Minor changes to the empty report:
    + Deleted some of the to:/cc: fields so it's even shorter
    + Deleted some white space, added "Section Header" so it's clear where to begin the writing.
    + Added a section header for "Reproducibility Software Information" since there is no longer a header for figures and tables.
    + Added a section header for References since this is still reading in the bibliography file.

Let me know if you agree with the changes (I can always revert the commits).

wfulp commented:
> Hi Jimmy, thanks for making this feature!
> 
> I made a couple of changes:
> 
> 1. Updated vignette so it describes how to use the empty report.
> 2. Minor changes to the empty report:
>    
>    * Deleted some of the to:/cc: fields so it's even shorter
>    * Deleted some white space, added "Section Header" so it's clear where to begin the writing.
>    * Added a section header for "Reproducibility Software Information" since there is no longer a header for figures and tables.
>    * Added a section header for References since this is still reading in the bibliography file.
> 
> Let me know if you agree with the changes (I can always revert the commits).

Yeah these changes are all great. Thanks
